src/shared/utils/index.ts

```diff
@@ -138,3 +139,31 @@ export const labelOf = (string: string) => {
 };
 
 export const isBrowser = typeof window !== 'undefined';
+
+/**
+ * Returns the logout URL of the realm the user is authenticated with
+ *
+ * @param identities
+ * @param realms
+ */
+export function getLogoutUrl(
+  identities: Identity[],
+  realms: { label: string; endSessionEndpoint: string }[]
+): string {
+  // find authenticated Identity and get realm name
+  const auths = identities.filter(
+    identity => identity['@type'] === 'Authenticated'
```

Comment:
"Authenticated" is a magic string, should be a constant or a config item, imported from a central storage place for these kinds of values

Reply:
It's not, it's a UNION type. You get a compile error if you don't use the right values.

Reply:
https://github.com/BlueBrain/nexus-sdk-js/blob/master/src/ACL/types.ts#L3